vagrant/boxes.d/00-centos.yaml

```diff
@@ -1,4 +1,14 @@
 boxes:
+  centos8-stream:
```

Comment:
Any reason you're adding centos8-stream back?

Reply:
yes, me failing at git, why else?